src/interpreter/ByteCode.h

```diff
@@ -188,7 +188,8 @@ class ByteCode {
     ByteCodeLOC m_loc;
     Opcode m_orgOpcode;
 
-    void dumpCode(size_t pos);
+    void dumpCode(size_t pos, char* code);
+    int dumpJumpPosition(size_t pos, char* code);
```

Comment:
Its just my suggestion. Could you change each char type pointer parameter to pointer to constant char (`char*` -> `const char*`) ?